patch/kernel/rockchip64-current/general-resolve-regulator-before-create.patch

```diff
@@ -0,0 +1,38 @@
+This patch reverts https://github.com/torvalds/linux/commit/aea6cb99703e17019e025aa71643b4d3e0a24413
```

Comment:
I believe this one is not needed anymore (and probably does not apply cleanly).
The main cause was fixed in 5.9.3 IIRC and I removed it with: https://github.com/armbian/build/commit/8718f57f9459b6a847037751e1cb2c7ce507878f